website/src/content/docs/docs/howtos/Generate client libraries/04renaming.mdx

```diff
@@ -20,8 +20,34 @@ By default, any language code generator will assume the TYPESPEC name is the cli
 <ClientTabs>
 
 ```typespec title=main.tsp
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using TypeSpec.Versioning;
```

Comment:
@lirenhe The code is missing import and using information maybe we don't need to update them since they are template code?